src/states_screens/credits.cpp

```diff
@@ -205,7 +206,11 @@ void CreditsScreen::loadedFromFile()
         // translations should be just before the last screen
         m_sections.swap( m_sections.size() - 1, m_sections.size() - 2 );
     }
-
+#ifdef __ANDROID__
```

Comment:
The special case for Android is ok, but there should also be a similar check to disable the button for unsupported platforms. LinkHelper::isSupported() or somethinG like that

This is the donate button, but same thing for the reset password button